### ISSUE TITLE
PyCon US 2018!

### ISSUE THREAD

**Issue by tannewt** [adafruit/circuitpython#781]
@dhalbert, @kattni and I will be attending PyCon US 2018. Lets plan events here. We also have a dedicated [Discord](https://adafru.it/discord) channel, #pycon-us.

I ordered 500 business cards earlier today:
![circuitpython_front_card](https://user-images.githubusercontent.com/52649/39330021-2030b99a-4955-11e8-980d-435ce7af91bc.jpg)
![circuitpython_back_card](https://user-images.githubusercontent.com/52649/39330027-230c9bd4-4955-11e8-8c71-de3b579e060f.jpg)

Please post other PyCon related plans here.

**Comment by kattni** [comment]
We had a discussion today about what projects we want to have available. The plan is to have three tables with printouts setup and the projects are: capacitive touch, NeoPixels, and CPU temperature logging. Only one requires external HW, and we'll have that covered. 

I've ordered USB cables, alligator clips and NeoPixels. 

Many people will have USB cables already, but with Apple and USB-C formats becoming more prevalent, we wanted to have something available in case anyone didn't have a micro-USB option available. 

I will be soldering alligator clips to the NeoPixels so they're ready to use easily with the Gemmas. We're going to have 8 available for use (2 rings, 2 jewels, and 4 sticks - which was decided by stock levels, not a specific reason on our part). This will be the next step in preparation once the order arrives (likely next week).

**Comment by tannewt** [comment]
Business cards shipped and should be to my place on Thursday.

**Comment by kattni** [comment]
Ordered Adabox 006 for the PyLadies auction, as well as NeoPixel rings as they were in stock. Will arrive tomorrow. Alligator clips and USB cables are already here. Prep will likely be happening next week.

**Comment by kattni** [comment]
Adabox 006 arrived today with the rest of the stuff we ordered for PyCon. Next week I'll be getting everything ready!

**Comment by kattni** [comment]
Today I hacked up a bunch of alligator clips, stripped and tinned the cut ends, fed them through shrink wrap, and soldered them to eight NeoPixel rings.


![img_20180508_001019182 2](https://user-images.githubusercontent.com/7865090/39737696-b3f04bb6-5254-11e8-96fa-ca01fdc1b883.jpg)

They're now packed up with the rest of the stuff we plan on bringing with us. So that part is ready to go!

**Comment by dhalbert** [comment]
1. I am experimenting with making single-page (probably two-sided) hardcopy cheat sheet, summarizing a bunch things and including the basic examples that will work on the Gemma.

2. Also may make up a gemma-pycon.uf2, with 2.3.1 and a filesystem containing stuff similar to what's shipped from the factory. Then people could update or restore easily.

**Comment by kattni** [comment]
PyCon was an absolute blast and a complete success. Our Open Spaces were full and people were incredibly engaged. The sprints resulted in tons of updates to libraries, core code and documentation, and continued after we had already departed the conference. We gave the NeoPixel rings and other hardware we had to individuals who were absolutely excited to have them. One of the best moments was when people came to us with new ideas and projects outside the quickstart we created and were so incredibly excited to share with us. I know we'll still manage to improve on it next year.

Thanks everyone!
